Source/WebCore/loader/DocumentLoader.h

```diff
@@ -780,6 +791,7 @@ class DocumentLoader
     ShouldOpenExternalURLsPolicy m_shouldOpenExternalURLsPolicy { ShouldOpenExternalURLsPolicy::ShouldNotAllow };
     PushAndNotificationsEnabledPolicy m_pushAndNotificationsEnabledPolicy { PushAndNotificationsEnabledPolicy::UseGlobalPolicy };
     InlineMediaPlaybackPolicy m_inlineMediaPlaybackPolicy { InlineMediaPlaybackPolicy::Default };
+    WebpagePreferences m_preferences;
```

Comment:
Why does DocumentLoader track `WebpagePreferences` instead of just `UserContentProvider`?